Modules/md5module.c

```diff
@@ -318,21 +318,28 @@ md5_done(struct md5_state *md5, unsigned char *out)
  * ------------------------------------------------------------------------
  */
 
-static PyTypeObject MD5type;
+typedef struct {
+    PyTypeObject* md5_type;
+} MD5State;
 
+static inline MD5State* md5_get_state(PyObject *module) {
```

Comment:
PEP 7 coding style:

```suggestion
static inline MD5State*
md5_get_state(PyObject *module)
{
```

Please apply the same change to other newly added functions.